source/agg-pipeline-builder/export-pipeline-to-language.txt

```diff
@@ -6,18 +6,74 @@ Export Pipeline to Specific Language
 
 .. default-domain:: mongodb
 
-*New in version 1.15.0*
-
 You can use the :ref:`Aggregation Pipeline Builder
-<compass-agg-builder>` to export your finished pipeline to one of the
-supported languages; Java, Node, C#, and Python 3. This feature
-facilitates formatting and exporting pipelines created in |compass| for
-use in your applications.
+<compass-agg-builder>` to export your finished pipeline to a language of your 
+choice to use in your application. |compass-short| facilitates the formatting 
+and exporting of pipelines created with Aggregation Pipeline Builder.
```

Comment:
[simplification]

I think we can combine these two sentences into one simpler one. We could try something like:

"You can use the Agg Pipeline Builder to format and export finished pipelines. You can export pipelines to a chosen language to use in your application."